packages/registry/test/fixtures/component-fixtures.tsx

```diff
@@ -9,19 +9,23 @@ export const TestComp: React.FunctionComponent<ITestProps> = (props: ITestProps)
 };
 
 TestComp.displayName = 'TestComp';
+Object.freeze(TestComp);
```

Comment:
Why do you need to freeze them?

Reply:
Becuase I saw that tests modify the display name:
```
TestComp.displayName = "ReturnDisplayName".
```

This means that one test affects other tests.

Reply:
Ah, ok. But tests shouldn't modify displayName, that's a mistake.